packages/amplify-provider-awscloudformation/src/system-config-manager.ts

```diff
@@ -154,9 +163,13 @@ async function getRoleCredentials(context: $TSContext, profileName: string, prof
   }
 
   return roleCredentials;
-}
+};
 
-async function getMfaTokenCode() {
+const getMfaTokenCode = async (context): Promise<string> => {
```

Comment:
I think `context` needs a type.
```suggestion
const getMfaTokenCode = async (context: $TSContext): Promise<string> => {
```